release-configs/qubes-os-iso-full-online.conf

```diff
@@ -0,0 +1,21 @@
+# vim: ft=make
+
+VERBOSE ?= 2
+
+BACKEND_VMM ?= xen
+
+GIT_BASEURL ?= https://github.com
+GIT_PREFIX ?= QubesOS/qubes-
+
+DIST_DOM0 ?=
+DISTS_VM ?=
+
+COMPONENTS = \
+    linux-yum \
+    builder-rpm
```

Comment:
`installer-qubes-os` component is missing here

Reply:
At first, it's normal. When adding the """installer-qubes-os"""", doing a """make iso""" it complains about not able to link all installer packages. Should I override components?

Reply:
Without `installer-qubes-os` here, it wont be downloaded at `make get-sources`, and consequently `make iso` will fail. In case of R4.1, installer components shouldn't be an issue anymore, as those are separate, but for R4.0 indeed it will demand installer components.
I think the easiest solution is to build them. But if you want to avoid that, please add a comment that user needs to download `installer-qubes-os` separately (like `make COMPONENTS=installer-qubes-os get-sources`).

Reply:
In fact this is what i'm doing in: https://github.com/fepitre/qubes-installer-qubes-os/blob/gitlab-build-latest/.gitlab-ci.yml
But I prefer to add the ```installer-qubes-os``` in the conf and adjust in the .gitlab-ci.yml. Ok with that?